src/BaseApiAbstract.php

```diff
@@ -413,7 +413,7 @@ private function processErrors(Response $response)
     /**
      * @throws SmartlingApiException
      */
-    private function processError(ResponseInterface $response): void
+    private function processError(Response $response): void
```

Comment:
`ResponseInterface` doesn't have `getBody` definition. Here must be `Response` class

Reply:
It's in \GuzzleHttp\Psr7\MessageTrait::getBody, that is used by \GuzzleHttp\Psr7\Response, so this change is probably not needed

Reply:
That said, I didn't check if it is in Guzzle 6

Reply:
Same for G6